lib/database/subscription.php

```diff
@@ -185,3 +180,118 @@ function mergeSubscribers(array $subscribersA, array $subscribersB): array
 
     return $subscribersA;
 }
+
+function getSubscribersOfGameWall($gameID): array
+{
+    return getSubscribersOfArticle(1, $gameID, (1 << 1));
```

Comment:
I realize these are just getting moved around, but it would be a good time to replace the constants with enums:
```
return getSubscribersOfArticle(ArticleType::Game, $gameID, 
    (1 << UserPreference::EmailOn_AchievementComment));
```
You could even just pass the UserPreferenceValue and do the shift inside the function.

Reply:
I've looked into that but went down a rabbit hole. I'll open a separate PR to clean up the subscriptions.